pkg/backend/store.go

```diff
@@ -54,18 +54,19 @@ type store struct {
 	rev    revision.Revisioner
 	config *config.Config
 	t      *storage.Table
-	client storage.TableServiceClient
 }
 
 func NewBackend(c *config.Config) (Backend, error) {
-	revisioner := revision.NewRevisioner(c.Runtime.StorageTable)
+	revisioner := revision.NewRevisioner(c.Runtime.RevisionStorageTable)
 	s := &store{
 		config: c,
 		rev:    revisioner,
 		t:      c.Runtime.StorageTable,
-		client: c.Runtime.TableClient,
 	}
 
+	if err := revisioner.EnsureRevStore(); err != nil {
```

Comment:
Can we move this to `NewRevisioner(..)` and make it return an error. The idea is to keep the details inside revisioner.

Reply:
Also remove it from `revisioner` interface

Reply:
thanks for the review, done!